src/transformers/models/oneformer/image_processing_oneformer.py

```diff
@@ -990,6 +996,24 @@ def encode_inputs(
             encoded_inputs["class_labels"] = class_labels
             encoded_inputs["text_inputs"] = text_inputs
 
+        def get_input_ids(task_input: str) -> List[int]:
+            if task_input == "semantic":
+                # this task is semantic
```

Comment:
ultra nit: picky but I don't think these comments are needed in this function

Reply:
Oh the comment means that those ids are what is outcome of `tokenizer.encode("this task is semantic")`.

I'll update the first comment to make it clearer.

Reply:
Yes, the comment is just paraphrasing the code in this case ;-)